src/letsbuilda/pypi/async_client.py

```diff
@@ -17,10 +17,21 @@ class PyPIServices:
     PACKAGE_UPDATES_FEED_URL: Final[str] = "https://pypi.org/rss/updates.xml"
 
     def __init__(self: Self, http_client: AsyncClient) -> None:
-        self.http_client = http_client
+        self.http_client: AsyncClient = http_client
```

Comment:
This is already typed right above here.
```suggestion
        self.http_client = http_client
```